projects/client/src/routes/search/+page.svelte

```diff
@@ -64,50 +74,38 @@
 
   <SearchModeToggles />
 
-  {#if $results}
-    {#if $results.type === "media"}
-      <SectionList
-        id="search-grid-list-movies"
-        title={m.text_search_results_for({
-          query: m.translated_value_type_movie(),
-        })}
-        items={$results.items.movies}
-        --height-list={mediaListHeightResolver("portrait")}
-      >
-        {#snippet item(result)}
-          <DefaultMediaItem type={result.type} media={result} style="cover" />
-        {/snippet}
-      </SectionList>
-
-      <SectionList
-        id="search-grid-list-shows"
-        title={m.text_search_results_for({
-          query: m.translated_value_type_show(),
-        })}
-        items={$results.items.shows}
-        --height-list={mediaListHeightResolver("portrait")}
-      >
-        {#snippet item(result)}
-          <DefaultMediaItem type={result.type} media={result} style="cover">
-            {#snippet tag()}
-              <InfoTag>
-                {toTranslatedValue("type", result.type)}
-              </InfoTag>
-            {/snippet}
-          </DefaultMediaItem>
-        {/snippet}
-      </SectionList>
-    {/if}
-
-    {#if $results.type === "people"}
-      <PersonList
-        title={m.text_search_results_for({
-          query: m.text_search_mode_people(),
-        })}
-        people={$results.items}
-      />
+  <div class="trakt-search-results-container">
+    {#if $results}
+      {#if $results.type === "media"}
+        <SectionList
+          id="search-grid-list-media"
+          title={m.list_title_search_results()}
+          items={$results.items}
+          --height-list="var(--height-poster-list)"
+        >
+          {#snippet badge()}
+            <TypeToggles types={selectedTypes} />
+          {/snippet}
+
+          {#snippet item(result)}
+            <DefaultMediaItem
+              type={result.type}
+              media={result}
+              style="cover"
+              tag={result.type === "show" ? showTag : undefined}
+            />
+          {/snippet}
+        </SectionList>
```

Comment:
Let's turn this into a `GridList` we have a lot of unused real-estate (let's keep display as posters).